src/models/generated/FullMachine.jl

```diff
@@ -99,14 +99,14 @@ function FullMachine(::Nothing)
         R_f=0,
         R_1d=0,
         R_1q=0,
-        L_d=1.0,
-        L_q=1.0,
-        L_ad=2.0,
-        L_aq=2.0,
-        L_f1d=1.0,
-        L_ff=2.0,
-        L_1d=1.0,
-        L_1q=1.0,
+        L_d=1,
+        L_q=1,
+        L_ad=2,
+        L_aq=2,
+        L_f1d=1,
+        L_ff=2,
+        L_1d=1,
+        L_1q=1,
```

Comment:
This is an error.  I used `jq` to format the JSON and it converted 2.0 to 2.  I'll fix it